build/gulpfile.scan.js

```diff
@@ -46,8 +46,7 @@ BUILD_TARGETS.forEach(buildTarget => {
 	if (platform === 'win32') {
 		tasks.push(
 			() => electron.dest(destinationPdb, { ...config, platform, arch: arch === 'armhf' ? 'arm' : arch, pdbs: true }),
-			util.rimraf(path.join(destinationExe, 'swiftshader')),
```

Comment:
These symbols don't seem to show up anymore in the first place, ref https://dev.azure.com/monacotools/a6d41577-0fa3-498e-af22-257312ff0545/_apis/build/builds/234677/logs/29

Reply:
`swiftshader` => `vk_swiftshader` and it has corresponding dll published seems fine to remove. However we don't publish pdb for `d3dcompiler_47.dll`, should it not be removed from the exe destination to exclude from scannning ? If the Binskim task has a exclude pattern, that would be better alternative as well.